Delivering cli scripts would mean an integration of additional
dependencies, which is not necessary at this point.

Not intended for upstream.

--- userpath-1.9.0/pyproject.toml
+++ userpath-1.9.0/pyproject.toml
@@ -28,9 +28,6 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = [
-  "click",
-]
 dynamic = ["version"]
 
 [project.urls]
@@ -39,9 +36,6 @@ History = "https://github.com/ofek/userp
 Issues = "https://github.com/ofek/userpath/issues"
 Source = "https://github.com/ofek/userpath"
 
-[project.scripts]
-userpath = "userpath.cli:userpath"
-
 [tool.hatch.version]
 path = "userpath/__init__.py"
 
